src/components/Form/Form.tsx

```diff
@@ -65,6 +67,30 @@ const Form = <T extends FormDetails>({
         type: "set_errors",
         errors: { ...state.errors, ...errors },
       });
+    } else {
+      setIsDirty(false); // Reset isDirty after successful submit
```

Comment:
Make sure to remove the comments you have added 👍

Reply:
okay I will remove all comments added by me.